.github/workflows/javabuild.yaml

```diff
@@ -0,0 +1,16 @@
+name: Java Build
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Build with Maven
+        run: mvn package --file step/portfolio/pom.xml
```

Comment:
I guess it's just portfolio/pom.xml. Presubmit check didn't find the file as well.

Reply:
Yes, I have changed that

Reply:
But I think there is still something that does not work well